llvm/test/tools/sycl-post-link/spec-constants/spec_const_and_split.ll

```diff
@@ -21,25 +21,27 @@
 
 declare dso_local spir_func zeroext i1 @_Z33__sycl_getScalarSpecConstantValueIbET_PKc(i8 addrspace(4)*)
 
-define dso_local spir_kernel void @KERNEL_AAA() {
+define dso_local spir_kernel void @KERNEL_AAA() #0 {
```

Comment:
I'm curious, why do we need to add the attribute to this test. Why It wasn't here before? Is it not possible to process the code without the attribute?

Reply:
If I understand correctly, this is how we detect whether the function should be considered as an entry point or not

Reply:
That's correct.